Wbooks/Localization/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,16 @@
+/* 
+  Localizable.strings
+  Wbooks
+
+  Created by Juan Martín Gordo on 12/04/2021.
+  
+*/
+
+/* LOGIN VIEW */
+"LOGIN_BUTTON" = "Log in";
+"USERNAME_LABEL" = "Username";
+"USERNAME_TEXT_FIELD_PLACEHOLDER" = "Enter a username";
+"PASSWORD_LABEL" = "Password";
```

Comment:
Looks good!
If you want to organize the strings better, you can use a prefix to indicate the view where the string is located. For instance:

```Swift
"LOGIN_VIEW.LOGIN_BUTTON" = "Log in";
"LOGIN_VIEW.USERNAME_LABEL" = "Username";
```

Also, capitalization is not necessary. If you want, you can write it like so:
```Swift
"loginView.loginButton" = "Log in";
"loginView.usernameLabel" = "Username";
```